app/helpers/selections_helper.rb

```diff
@@ -1,2 +1,50 @@
+require 'json'
+require 'ostruct'
+
 module SelectionsHelper
+
+  def get_forecast(stations)
+
+    w_api = Wunderground.new("dda1e42fba07eb1c")
```

Comment:
The api key belongs in a config variable and accessed here with `Rails.configuration.wunderground_key`.